app/assets/stylesheets/ui_components.scss.erb

```diff
@@ -7,3 +7,12 @@
 @import 'bootstrap-chosen';
 
 @import 'ui_components/*';
+
+<% UiComponents::Engine.components_paths.each do |path| %>
+  <%
```

Comment:
Apparently not. I suppose `require_asset` works more procedurally than functional. I will check, though, if it still works in production mode... :D